src/Update/Program.cs

```diff
@@ -24,13 +25,19 @@ enum UpdateAction {
 
     class Program : IEnableLogger 
     {
+        private static int ERROR_EXE_MACHINE_TYPE_MISMATCH = 216;
         static OptionSet opts;
 
         public static int Main(string[] args)
         {
             var pg = new Program();
             try {
                 return pg.main(args);
+            } catch (AggregateException ex) {
+                Console.Error.WriteLine(ex);
+                return ex.InnerExceptions
```

Comment:
I *think* ex.InnerExceptions.OfType<Win32Exception>().Any( blah blah) is collectively a boolean. That is one of the ugliest ternary operators I've ever seen. Could we store the boolean result in a variable and use that in the ternary, instead?

Reply:
Done